src/core/Akka.Remote.Tests/Transport/MultiTransportAddressingSpec.cs

```diff
@@ -87,11 +87,19 @@ public async Task Should_Use_Second_Transport_For_Communication()
             Shutdown(secondSystem);
         }
 
+        return;
+
         async Task PingAndVerify(string scheme, int port)
         {
             var selection = Sys.ActorSelection($"akka.{scheme}://MultiTransportSpec@localhost:{port}/user/echo");
-            selection.Tell("ping", TestActor);
-
+            
+            // important: https://github.com/akkadotnet/akka.net/issues/7378 only occurs with IActorRefs
+            var actor = await selection.ResolveOne(TimeSpan.FromSeconds(1));
+            
+            // assert that the remote actor is using the correct transport
+            Assert.Contains(scheme, actor.Path.Address.Protocol);
```

Comment:
Currently fails here - the `IActorRef` returned to the `ResolveOne` method belongs to transport 1 rather than transport 2.